src/components/ArticleList.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from "react"
+import { fetchArticleData } from "./api"
+import ArticleCard from "./ArticleCard"
+import DisplayBox from "./DisplayBox"
+import { Link } from "react-router-dom"
+
+function ArticleList() {
+
+    const [articles, setArticles] = useState([])
+
+    useEffect(() => {
+        fetchArticleData().then((articleData) => {
```

Comment:
This could be named more clearly. At the moment it's confusing to the user - is the data the object being returned, or a key on that object? Something more generic like 'response' would be good for whatever is being returned out of the promise.